tests/dft-grad-lr1/input.dat

```diff
@@ -0,0 +1,28 @@
+#! DF-BP86-D2 cc-pVDZ frozen core gradient of S22 HCN
+
+ref = psi4.Matrix.from_list([                                 #TEST
```

Comment:
Just out of interest, where's this reference gradient coming from?

Reply:
5 point stencil with a large grid showed that analytical and FD returned the same result to ~6 decimal places. I simply took the analytical at that point, im not sure of another code that has density fitted gradients for range-separated functionals. Probably Molpro is the best bet if anyone has access.